app/views/merchant_invoices/show.html.erb

```diff
@@ -1,6 +1,19 @@
 <h1>Invoice <%=@invoice.id%>'s Information</h1>
 <h3>Merchant: <%=@merchant.name%></h3>
 <p>Status: <%=@invoice.status%></p>
-<p>Status: <%=@invoice.status%></p>
 <p>Created On: <%=@invoice.created_at.strftime("%A, %B %d, %Y")%></p>
 <p>Customer: <%=@invoice.customer_name%></p>
+
+<h2><%=@invoice.id%>'s Items</h2>
+<%@invoice.items.each do |item_1|%>
+  <%@invoiced_item.each do |item_2|%>
+<% if item_1.merchant_id == @merchant.id && item_2.item_id == item_1.id%>
+<section id="invoice-#{item.id}-info">
+  <p><%=item_1.name%></p>
+  <p>Quantity Ordered: <%=item_2.quantity%></p>
+  <p>Unit Price: $<%=(item_2.unit_price.to_f/100)%></p>
+  <p>Status: <%=item_2.status%></p>
+</section>
+<%end%>
+<%end%>
+<%end%>
```

Comment:
Saving as a reminder to indent

Reply:
DONT TELL ME MY BUSINESS.

I'll fix that :D